twake/backend/node/src/services/channels/web/controller.ts

```diff
@@ -2,14 +2,28 @@ import { Channel } from "../entities";
 import ChannelServiceAPI from "../provider";
 import { CreateChannelBody } from "./types";
 
+
 export default class ChannelController {
-  constructor(private service: ChannelServiceAPI<Channel>) {}
+  constructor(private service: ChannelServiceAPI<Channel>) { }
 
   async create(channel: CreateChannelBody): Promise<Channel> {
     const entity = new Channel();
     // TODO: Create the Channel entity from the CreateChannelBody
     // The CreateChannelBody is already validated by the web framework
+
```

Comment:
I introduced class-transformer in some part of my code, going to push a PR so that we can reuse it and avoid this